api/python/cellxgene_census/tests/experimental/pp/test_stats.py

```diff
@@ -88,6 +88,62 @@ def test_mean_variance(
                 assert np.allclose(variance, mean_variance["variance"], atol=1e-5, rtol=1e-2)
 
 
+@pytest.mark.experimental
+@pytest.mark.live_corpus
+@pytest.mark.parametrize("axis", [0])
+@pytest.mark.parametrize("calc_mean,calc_variance", [(True, True)])
+@pytest.mark.parametrize(
+    "experiment_name,obs_value_filter,obs_coords",
+    [
+        ("mus_musculus", 'tissue_general == "liver" and is_primary_data == True', ()),
+        pytest.param("mus_musculus", "is_primary_data == True", (slice(0, 400_000),), marks=pytest.mark.expensive),
+        pytest.param("homo_sapiens", "is_primary_data == True", (slice(0, 400_000),), marks=pytest.mark.expensive),
+    ],
+)
+def test_mean_variance_exclude_zeros(
```

Comment:
The axis=1 was an omission, thanks for catching that! I also agree it's probably better to merge the tests into one, kinda like it did in `test_online`. It will require some additional branching but it will also make it faster, since it won't have to re-query the census twice. Will go for it.

Reply:
Update: since this test requires materializing the matrix in memory to calculate mean/var without the zeros, it also requires way more memory. Therefore, I had to keep it separate and change the query strategy so that the matrix doesn't get too big. I also added back both axis.